resources/build/conanfile.txt

```diff
@@ -3,6 +3,10 @@
 cpython/3.9.7
 # Same as ASWF CY2022 Docker image: https://github.com/AcademySoftwareFoundation/aswf-docker/blob/master/ci-base/README.md
 pybind11/2.8.1
+# Unit tests
+# TODO(DF): Make this conditional (likely requires conanfile.py)
+catch2/2.13.8
+trompeloeil/42
```

Comment:
[fix] Can we add to the commit message the reason for picking `catch2` and `trompeloeil` over the alternatives?

Reply:
Added a commit (with some dummy changes) with new commit message in b6acc424